aws/ec2-instances/outputs.tf

```diff
@@ -54,6 +54,21 @@ output "rhel9_cnspec" {
   value = module.rhel9_cnspec.public_ip == null ? "" : "ssh -o StrictHostKeyChecking=no -i ~/.ssh/${var.aws_key_pair_name} ec2-user@${module.rhel9_cnspec.public_ip}"
 }
 
+# ubuntu2004
+output "ubuntu2004" {
+  value = module.ubuntu2004.public_ip == null ? "" : "ssh -o StrictHostKeyChecking=no -i ~/.ssh/${var.aws_key_pair_name} ubuntu@${module.ubuntu2004.public_ip}"
+}
+
+output "ubuntu2004_cnspec" {
+  value = module.ubuntu2004_cnspec.public_ip == null ? "" : "ssh -o StrictHostKeyChecking=no -i ~/.ssh/${var.aws_key_pair_name} ubuntu@${module.ubuntu2004_cnspec.public_ip}"
+}
+
+output "ubuntu2004_cis_cnspec" {
+  value = module.ubuntu2004_cis_cnspec.public_ip == null ? "" : "ssh -o StrictHostKeyChecking=no -i ~/.ssh/${var.aws_key_pair_name} ubuntu@${module.ubuntu2004_cis_cnspec.public_ip}"
+}
+
```

Comment:
ubuntu2004_cis is missing in the outputs

Reply:
fixed